src/ui/pages/payment-entry-page.svelte

```diff
@@ -163,10 +162,11 @@
   >
     <div class="rc-checkout-form-container" hidden={!!modalErrorMessage}>
       <div class="rc-payment-element-container">
-        <StripePaymentElements
+        <StripeElements
           bind:submit={stripeSubmit}
           bind:confirm={stripeConfirm}
           bind:stripeLocale
+          {priceBreakdown}
```

Comment:
Since we're passing `priceBreakdown`, lets remove the field `taxCollectionEnabled`